sagemaker-pyspark-sdk/src/sagemaker_pyspark/wrapper.py

```diff
@@ -157,3 +159,27 @@ def _to_java(self):
 
     def _from_java(self):
         raise NotImplementedError()
+
+
+# wrap scala.collection.immutable.List
+class ScalaList(SageMakerJavaWrapper):
+
+    def __init__(self, p_list):
+        self.p_list = p_list
+
+    def _to_java(self):
+        # Since py4j cannot deal with scala list directly
+        # we convert to scala listmap as an intermediate step
+        s_list = self._new_java_obj("scala.collection.immutable.ListMap")
+        key = 0
+        for elem in self.p_list:
```

Comment:
you can replace the need for manually managing key by doing
`
for key, elem in enumerate(self.p_list):`

Reply:
Now use zip() for better codestyle